__tests__/integration.test.js

```diff
@@ -285,3 +285,26 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  it("GET:200 should return all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        });
+      });
+  });
+
+  it("GET:404 should display error when route does not exist", () => {
+    return request(app)
+      .get("/api/notARoute")
+      .expect(404)
```

Comment:
you may have a similar test to this from earlier with topics??

Reply:
duplicate test removed